tailcfg/tailcfg.go

```diff
@@ -1347,9 +1342,9 @@ type MapResponse struct {
 	// ControlTime, if non-zero, is the current timestamp according to the control server.
 	ControlTime *time.Time `json:",omitempty"`
 
-	// TKA, if non-nil, describes updates for the local instance of the
-	// tailnet key authority.
-	TKA *TKAMapResponse `json:",omitempty"`
+	// TKAInfo, if non-nil, describes controls' view of tailnet key
+	// authority state.
```

Comment:
Document what nil means for both the initial MapResponse and on streams ones, where zero for all these fields means "no change from before".

For the initial MapResponse, is a nil TKAInfo equivalent to a pointer to the zero value?